specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json

```diff
@@ -2348,7 +2348,7 @@
           "description": "The reference to the VpnClientConfiguration resource which represents the P2S VpnClient configurations."
         },
         "bgpSettings": {
-          "$ref": "#/definitions/BgpSettings",
+          "$ref": "#/definitions/VirtualNetworkGatewayBgpSettings",
```

Comment:
This could be considered a breaking change in SDKs

Reply:
We are discussing about it. I will give update soon. Thanks @number213

Reply:
I have removed this breaking change and added properties in existing BgpSettings.

Reply:
Fixed this comment so resolving.